src/Alert/README.md

```diff
@@ -12,10 +14,17 @@ notes: |
 
 ---
 
-<p className="lead">
+The Alert component displays a short, important message in a way that attracts the user's attention.
+Alerts offer four severity levels that set a distinctive icon and color:
+- **Info**: used to convey general information or actions that aren't critical, note that info variant should not contain any icons in it
+- **Warning**: used to display information that needs attention
+- **Success**: used for success messages
+- **Danger**: used to communicate problems that have to be resolved immediately
+
+<p>
   This is a pass through component from React-Bootstrap.<br/>
   <a href="https://react-bootstrap.github.io/components/alerts/" target="_blank" rel="noopener noreferrer">
-    See React-Bootstrap for documentation.
+  See React-Bootstrap for additional documentation.
```

Comment:
Since this component adds some custom things on top of the base `Alert` component from react-boostrap, maybe instead of saying "This is a pass through component...",  we can be a bit more explicit by saying something along the lines of, "This component utilizes and extends the `Alert` component from react-bootstrap"? 🤔 

I think we should only say something is a pass through component if we use the vanilla component exported by react-bootstrap without any modifications. What do you think?

Reply:
I totally agree! I started thinking about that too while working on the next PR, updated phrasing here also, thanks